File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.java
public class Solution {
    public static boolean isPalindrome(String s) {
        StringBuilder st = new StringBuilder(s);
        return s.equals(st.reverse().toString());
    }

    public String firstPalindrome(String[] words) {
        String ans = "";
        for (int i = 0; i < words.length; i++) {
            if (isPalindrome(words[i])) {
                ans = words[i];
                break;
            }
        }
        return ans;
    }
}

File: 2108-find-first-palindromic-string-in-the-array/2108-find-first-palindromic-string-in-the-array.py
class Solution:
    def firstPalindrome(self, words: List[str]) -> str:
        def pal_check(s):
            if s==s[::-1]:
                return True
            else:
                return False
        for word in words:
            if pal_check(word)==True:
                return word
                break
        return ""
        
                    